docs/blueprints/EP036.rst

```diff
@@ -0,0 +1,226 @@
+:EP: 36
+:Title: Openflow multi-table pipeline processing
+:Authors:
+    - Aldo Ortega <aortegay AT fiu DOT edu>
+    - Vinicius Arcanjo <vindasil AT fiu DOT edu>
+    - Italo Valcy <idasilva AT fiu DOT edu>
+
+:Created: 2022-05-04
+:Updated: 2022-06-04
+:Kytos-Version: 2022.3
+:Status: Draft
+
+***************************************************
+EP36 - Openflow multi-table pipeline processing
+***************************************************
+
+Abstract
+========
+
+This blueprint will present the requirements for the implementation of switch multi-tables through new NApp called **multi_table**.
+
+Motivation
+==========
+
+Multi-table processing is desireable to take advantage of a pipelined processing of packets within a switch. It is necessary to ensure the consistency of tables to avoid missing packets thus a NApp will take care of the table system by being aware of all the flows created from NApps and users.
+
+Specification
+=============
+
+Define the table to be used by NApps and to be used by users which are going to be configurable.
+Also, a new NApp **multi_table** that will initially have the following responsibilities:
+
+  - Know which table is enabled.
+  - Ensuring the existence of a reasonable table-miss flow
+  - Publish events to handle the assigned table of each NApp
+  - In charge of modifying and analyzing **pipelines** collection
+  - The cookie prefix is **0xAD** for its posted flows.

```

Comment:
Could you also add this point below (since it's part of what you specified when enabling a pipeline)? 
- Requesting flow deletions and installations on behalf of other NApps when a pipeline changes.